tests/apps/test_graph_similarity.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Unit tests for strawberryfields.apps.graph.similarity
+"""
+# pylint: disable=no-self-use,unused-argument,too-many-arguments
+import itertools
+
```

Comment:
Is it standard to use these spaces between import statements?

Reply:
Nice! Already in my pre-commit hook 😁